### DISCUSSION TITLE
Where v2.14?

### DISCUSSION THREAD

**Discussion by burner1024** [argoproj/argo-cd#20806] [Q&A]
I must be blind or something, but I'm just not seeing it.

![Captura desde 2024-11-15 14-21-55](https://github.com/user-attachments/assets/c930c738-4eb2-48a3-999b-ef247e676fcd)
![Captura desde 2024-11-15 14-23-41](https://github.com/user-attachments/assets/af36512a-23e2-4fd1-bd97-172c435e7f13)
![Captura desde 2024-11-15 14-24-09](https://github.com/user-attachments/assets/61542706-1931-49e7-8886-11594e77b478)
![Captura desde 2024-11-15 14-21-34](https://github.com/user-attachments/assets/7d7f77cf-8821-411f-9f7a-314c8e5b8443)

**Comment by rumstead** [comment]
The 2.14 release candidate hasn't been cut yet. The first stable, non-release candidate of [2.13 was just cut 2 weeks ago](https://github.com/argoproj/argo-cd/releases/tag/v2.13.0). 2.14 should be expected sometime in [February](https://argo-cd.readthedocs.io/en/stable/developer-guide/release-process-and-cadence/#release-process).

**Reply by crenshaw-dev** [reply]
SECURITY-INSIGHTS.md is an imperfect reflection of the state of the project. It's possible that we should change CI to not bump that version so soon, but I'm not sure it's worth the effort.

**Reply by rumstead** [reply]
TIL
